instance/models/mixins/openedx_config.py

```diff
@@ -66,6 +67,8 @@ def _get_configuration_variables(self):
                 "BROKER_HEARTBEAT": 0,
                 "ADDL_INSTALLED_APPS": [
                 ],
+                # Required for Safari auth with secure SameSite cookies
+                "DCS_SESSION_COOKIE_SAMESITE": "Lax",
```

Comment:
For reference. I also left a comment at SE-3154 about `Lax` vs. `None`; we can discuss it there.

Reply:
@clemente I've read your notes, and since I don't know what the full effects of this change might be, I'm going to remove this line, and we can look into it more fully if clients report issues with Safari.

[`SameSite=Lax` is the default for the devstack](https://github.com/edx/edx-platform/blob/e7099518d92dddbcc671b014045cde800d23f2d1/lms/envs/devstack.py#L400), which indicated to me that this was a relatively safe option, but I haven't tested every case, so can't be sure.

I found that it fixed SSO logins on Safari for my client, so have put this into their config, but maybe it's something particular to their setup?